### PR TITLE
Blacken CyHy report code

### PR DESCRIPTION
**Pull request by dav3r**
# 

## 🗣 Description
This PR blackens the CyHy report generation code, in preparation for future changes.

## 💭 Motivation and Context
Cleaner code is happier code.

## 🧪 Testing
Verified that reports can still be generated successfully.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
I looked through these changes, and they all look safe to me.

File: cyhy_report/customer/generate_report.py [hunk 1]
```diff
@@ -67,61 +68,67 @@
...
 SNAPSHOT_HISTORY_LIMIT = 30
 MAX_REPORTCARD_BOX_DISPLAY = 5000
-CRITICAL_AGE_OVER_TIME_CUTOFF_DAYS = 365    # Determines how far back we look at tickets for critical-vuln-ages-over-time figure
-TICKET_AGE_BUCKET_CUTOFF_DAYS = 30  # Dividing line between 'young' and 'old' tickets in Cyber Exposure-esque graphs
-ACTIVE_CRITICAL_AGE_CUTOFF_DAYS = 180 # Max number of days to display in __figure_active_critical_vuln_age_distribution()
-ACTIVE_CRITICAL_AGE_BUCKETS = [(0,7), (7,14), (14,21), (21,30), (30,90)]
+CRITICAL_AGE_OVER_TIME_CUTOFF_DAYS = 365  # Determines how far back we look at tickets for critical-vuln-ages-over-time figure
+TICKET_AGE_BUCKET_CUTOFF_DAYS = (
+    30  # Dividing line between 'young' and 'old' tickets in Cyber Exposure-esque graphs
+)
```
Comment (mcdonnnj) [inline cyhy_report/customer/generate_report.py]: Should we kick the comments to the preceding line to improve readability?

**Review by mcdonnnj — APPROVED** [review]
LGTM ✅
[blacken all the code!](https://www.youtube.com/watch?v=pAgnJDJN4VA)

File: cyhy_report/customer/generate_report.py (hunk 1 above)
Comment (dav3r) [reply cyhy_report/customer/generate_report.py]: Good idea- see https://github.com/cisagov/cyhy-reports/pull/55/commits/d6863789f917981fbebbd3c859086f39fe57cf0b.

**Pull request merged**